include/deal.II/base/tensor.h

```diff
@@ -24,21 +24,12 @@
 #include <deal.II/base/template_constraints.h>
 #include <deal.II/base/tensor_accessors.h>
 
+#include <Kokkos_Core.hpp>
+
```

Comment:
```suggestion
#include <Kokkos_Array.hpp>

```

Can we get away with using that instead?